### PR TITLE
[task #1] Add Charlotte Page

### PR DESCRIPTION
**Pull request by charlotteds**

### CODE REVIEW

File: charlotte.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Charlotte's Page</title>
+    </head>
+    <body>
+        <h1>Charlotte</h1>
```
Comment (inahsanjuan) [inline charlotte.html]: I dont like your name.
Can you change to Charlie

**Review by inahsanjuan — CHANGES_REQUESTED** [review]
Pls check my comments

File: charlotte.html (hunk 1 above)
Comment (charlotteds) [reply charlotte.html]: @inahsanjuan 
Comment is only on line 7.
Should I also update other occurrences of "Charlotte" (e.g. filename, titles).

Comment (inahsanjuan) [reply charlotte.html]: Yes pls!

**Review by inahsanjuan — APPROVED** [review]
LGTM!

**Pull request merged**